RPIPs/RPIP-39.md

```diff
@@ -81,7 +81,7 @@ The support incentive lever aims to influence the quality and consistency of the
 * This incentive SHOULD be split equally between bounty support contacts listed on a bounty where the following conditions are met:
   1. The bounty MUST be completed (in full or in part) to the satisfaction of the GMC.
   2. The contact MUST be publicly listed in the bounty definition of the completed bounty.
-  3. The contact MUST NOT be affiliated with the completing bounty hunter.
+  3. The contact MUST NOT have a significant conflict of interest between Rocket Pool and the completing bounty hunter.
```

Comment:
I liked your wording in https://github.com/rocket-pool/RPIPs/pull/128#discussion_r1499185660 better. I struggled to parse this sentence. I think I concluded that the premise is they must be ~neutral between RP and the bounty hunter?

Reply:
Updated. Thanks for pushing back on the issue.